tasks/main.yml

```diff
@@ -10,9 +10,11 @@
   file:
     path: "{{ homebrew_prefix }}"
     mode: 0775
+    owner: root
     group: admin
     state: directory
   become: yes
+  become_user: root
```

Comment:
That's fine. I had trouble testing on a remote machine over ssh, but I think that can be addressed later.